server/conf/application.conf

```diff
@@ -588,3 +588,6 @@ civiform_applicant_enabled_languages = ${?CIVIFORM_APPLICANT_ENABLED_LANGUAGES}
 # Typically, this value is 1.
 num_trusted_proxies = 1
 num_trusted_proxies = ${?NUM_TRUSTED_PROXIES}
+
+# Default to the local PNG logo
+civic_entity_small_logo_url = null
```

Comment:
(blocking) Sorry, probably didn't give enough detail in my original message. Doing this here will override the environment variables and will always be null now for any deployment. You need to edit the `whitelabel.conf` file which is imported into this base config.

What I think you want to do is set `civic_entity_small_local_url = null` [here](https://github.com/civiform/civiform/blob/afba63d9344776465884f7992a47bf2988e0e6f1/server/conf/helper/whitelabel.conf#L6C32-L6C130).

You'll need to make sure of the following though:

1) Local/deployed (staging) sites are indeed pulling from the file now instead of the url to githubusercontent as showing in the above link.

2) Verify that the image is loaded if the `CIVIC_ENTITY_SMALL_LOCAL_URL` is set. You can simulate that locally by setting the value [here](https://github.com/civiform/civiform/blob/afba63d9344776465884f7992a47bf2988e0e6f1/docker-compose.yml#L125) and running `bin/run-dev` and make sure it's now pointing back to the github link and not localhost.

Reply:
Alternatively just set `civic_entity_small_local_url = null` in `application.dev.conf` and dev, browser-tests, and unit tests only should pull from the localhost image. But you'll want to test it to be certain. You don't need to set it in the `application.dev-browser-tests.conf` or `application.test.conf` files since those inherit from `application.dev.conf`

Reply:
Thanks! Editing whitelabel.conf works and the other way doesn't. I tested locally.

I haven't deployed before, so I'll go through the instructions tomorrow to deploy to a staging environment. https://github.com/civiform/civiform/wiki/Deploy-System

Is doing a full deployment to test necessary?

Reply:
If it works testing locally we can just test it on staging after tomorrow's release. Just don't merge it until then that way we don't have to rush if we need to rollback.